streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -236,6 +238,16 @@ public static String safeUniqueTestName(final Class<?> testClass, final TestName
                 .replace('=', '_');
     }
 
+    public static String safeUniqueTestName(final Class<?> testClass, final TestInfo testInfo) {
```

Comment:
please add documentation

Reply:
We can refactor the existing `safeUniqueTestName` as follows to extract out common parts such as:

```

public static String safeUniqueTestName(final Class<?> testClass, final TestName testName) {
    return IntegrationTestUtils.safeUniqueTestName(testClass, testName.getMethodName())
}

public static String safeUniqueTestName(final Class<?> testClass, final TestInfo testInfo) {
    return IntegrationTestUtils.safeUniqueTestName(testClass, testInfo.getTestMethod().map(Method::getName))
}

private static String safeUniqueTestName(final Class<?> testClass, final String methodName) {
   ...
}
```

Reply:
Fair point, I will make the changes in the next commit.